{"uuid":"bb2f8e83-f6bd-406b-8331-39b82ab2f943","historyId":"90688c77927fc5b097e32bf39ac80028","fullName":"LoginTest.signInTest","labels":[{"name":"package","value":"LoginTest"},{"name":"testClass","value":"LoginTest"},{"name":"testMethod","value":"signInTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"discshop"},{"name":"subSuite","value":"LoginTest"},{"name":"host","value":"PL-PDW-98372"},{"name":"thread","value":"29188@PL-PDW-98372.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[{"name":"ID-002","type":"tms"}],"name":"signInTest","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Deliberately broken test for reporting demonstration expected [false] but found [true]","trace":"java.lang.AssertionError: Deliberately broken test for reporting demonstration expected [false] but found [true]\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\r\n\tat org.testng.Assert.assertFalse(Assert.java:67)\r\n\tat LoginTest.signInTest(LoginTest.java:22)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:567)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","description":"Login and navigation to Welcome screen","steps":[],"attachments":[{"name":"Failed test screenshot","source":"f60e4bff-c913-4261-bc37-e47786333793-attachment","type":"image/png"},{"name":"Browser information","source":"b342acc8-ab5e-49b7-a716-468399c2507e-attachment","type":"text/plain"}],"parameters":[],"start":1721101682241,"stop":1721101690877}